packages/edit-site/src/components/block-editor/index.js

```diff
@@ -169,7 +169,9 @@ export default function BlockEditor() {
 				{ ( [ editorCanvasView ] ) =>
 					editorCanvasView ? (
 						<div className="edit-site-visual-editor is-focus-mode">
-							<ResizableEditor enableResizing>
+							<ResizableEditor
+								enableResizing={ canvasMode !== 'view' }
+							>
```

Comment:
Hi @andrewserong 

I have a PR up that allows the consuming component to control the resizable editor:

- https://github.com/WordPress/gutenberg/pull/50682

So style book will be able to pass this down. I'm not sure whether it makes your life harder or easier 🤣 

But the logic is that, some use cases might want to force "resizeability" either way, e.g., the global styles revisions pane.

Reply:
Oh, thanks Ramon! It won't affect my work too much either way, but I like the approach in that PR. I'll review that shortly, and once it's in I can rebase this PR 🙂